wire-ios-sync-engine/Tests/Source/Integration/APNSTests.m

```diff
@@ -115,7 +118,9 @@ - (void)testThatItFetchesTheNotificationStreamWhenReceivingNotificationOfTypeNot
     XCTestExpectation *fetchingExpectation = [self expectationWithDescription:@"fetching notification"];
     
     __block NSUInteger requestCount = 0;
-    NSUUID *lastNotificationId = self.userSession.syncManagedObjectContext.zm_lastNotificationID;
+    LastEventIDRepository *lastEventIDRepository = [[LastEventIDRepository alloc] initWithUserID:self.currentUserIdentifier
+                                                                                    userDefaults:NSUserDefaults.standardUserDefaults];
```

Comment:
do we have to use standardUserdefaults here? or can we use an ephemeral one to avoid leaking data to other tests?

Reply:
fixed in c8a59a522c5b2a06d5a9bf871a2a7e174175247d, which uses a random user defaults for each test.